calendar_v17/GoogleMapsScraper.py

```
@@ -0,0 +1,1510 @@
+                
+            except Exception as e:
+                self.queue.put(('status', f"Error during scraping: {str(e)}"))
+                self.queue.put(('error', f"An error occurred during scraping: {str(e)}"))
+                
+            finally:
+                # Clean up
+                if driver:
+                    try:
+                        driver.quit()
+                        self.queue.put(('status', "Browser closed."))
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)